src/libs/actions/Policy.js

```diff
@@ -909,6 +909,40 @@ function buildOptimisticCustomUnits() {
     };
 }
 
+function createDraftInitialWorkspace(policyOwnerEmail = '', policyName = '', policyID = generatePolicyID()) {
```

Comment:
Looks like we have 2 times a default value for `policyName`. So, either call a fallback function right in arguments:
```suggestion
function createDraftInitialWorkspace(policyOwnerEmail = '', policyName = generateDefaultWorkspaceName(policyOwnerEmail), policyID = generatePolicyID()) {
```
Or remove a default value:
```suggestion
function createDraftInitialWorkspace(policyOwnerEmail = '', policyName, policyID = generatePolicyID()) {
```

Reply:
I just left it how it was

Reply:
Also works